modules/troubleshooting-network-observability-after-installation.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * networking/network_observability/troubleshooting-network-observability.adoc
+
+:_content-type: PROCEDURE
+[id="configure-network-traffic-console_{context}"]
+= Configuring network traffic menu entry in the {prouct-title} console
+
+Manually configure the network traffic menu entry in the {product-title} console where network traffic menu entry is not connected to the network.
```

Comment:
I feel we should rephrase the `where network traffic menu entry is not connected to the network.` part.

What about `when network traffic menu entry is not listed in administrator 'Observe' menu.` ?

Reply:
+1 to @jpinsonneau feedback.

Reply:
Addressed

Reply:
@jpinsonneau one doubt: the 'Observe' menu is present in the OpenShift console right?